yandex/structures.go

```diff
@@ -30,8 +25,11 @@ import (
 	ltagent "github.com/yandex-cloud/go-genproto/yandex/cloud/loadtesting/api/v1/agent"
 	"github.com/yandex-cloud/go-genproto/yandex/cloud/organizationmanager/v1"
 	"github.com/yandex-cloud/go-genproto/yandex/cloud/vpc/v1"
-
 	"github.com/yandex-cloud/terraform-provider-yandex/yandex/internal/hashcode"
+	"google.golang.org/genproto/googleapis/type/dayofweek"
+	"google.golang.org/genproto/googleapis/type/timeofday"
+	"google.golang.org/grpc"
+	proto "google.golang.org/protobuf/proto"
 )
 
 func IterateKeys(d *schema.ResourceData, key string) []string {
```

Comment:
This function is not used for `labels` processing.

One way to modify labels is to update   [expandLabels](https://github.com/yandex-cloud/terraform-provider-yandex/blob/master/yandex/structures.go#L96) 

But desire way is to have validation on this field, because other way users can meet undocumented behaviour.